src/main/java/com/stakater/nordmart/CartServiceApplication.java

```diff
@@ -20,6 +22,11 @@ public static void main(String[] args)
         SpringApplication.run(CartServiceApplication.class, args);
     }
 
+    @Bean
+    public KeycloakSpringBootConfigResolver keycloakConfigResolver() {
+        return new KeycloakSpringBootConfigResolver();
+    }
+
```

Comment:
Move this to a separate class `KeycloakConfiguration`

```
@org.springframework.context.annotation.Configuration
public class KeycloakConfiguration
{
 //your code for bean here
}
```

Reply:
Done.